UKHomeOffice/sgmr-service
# SPCR-198 Fix erroring issue on add person on person page

## Description
Add person on person page was not showing an error on the front end when a user typed a duplicate document number. This error appeared on the API, and just needed to be displayed on the site when the error was triggered. 

## To Test
- Go to the People page
- Click on Save a Person
- Type in the details of someone you have already saved, including their document number
- Click on Add to saved people list
- You should get an error stating that the document number entered already exists
- Clear the fields and type in the details of someone completely new
- Click on Add to saved people list
- You should be returned to the people page and be able to see the new person listed on the people table

## Developer Checklist

\* Required






Ensure you delete the branch once the Pull Request is merged.
